app/models/concerns/draft_timeliness.rb

```diff
@@ -0,0 +1,25 @@
+module DraftTimeliness
+  module ResponseAdded
+    extend ActiveSupport::Concern
+
+    def log_compliance_date
```

Comment:
Method should have exclamation mark `!` in name i.e. `log_compliance_date!` to ensure user of the method knows this method raises an exception on failure. Applies to both instances of `log_compliance_date`